{{ cookiecutter.repo_name }}/README.md

```diff
@@ -1,16 +1,75 @@
 # {{cookiecutter.project_name}}
 
-## Setup
+**_Change this to the full project name 👆_**
 
-- Meet the data science cookiecutter [requirements](http://nestauk.github.io/ds-cookiecutter/quickstart), in brief:
-  - Install: `direnv` and `conda`
-- Run `make install` to configure the development environment:
-  - Setup the conda environment
-  - Configure `pre-commit`
+**_Add a one-liner that describes this project._**
 
-## Contributor guidelines
+## 👋 Welcome!
 
-[Technical and working style guidelines](https://github.com/nestauk/ds-cookiecutter/blob/master/GUIDELINES.md)
+**_Write a longer description of the project here. Put what you like here, but you might want to include:_**
+
+- **_Some background on the project and what its aims are_**
+- **_A high level description of what the code in this repository does_**
+- **_Links to other relevant work or information on Nesta's website_**
+- **_The respository's development status. Is this currently under development, finished, parked (offer an explanation if you feel it is needed)._**
+- **_Link to any particularly important project outputs:_**
+  - **_reports_**
+  - **_slide decks_**
+  - **_media_**
+
+![Maybe add an image or nice graphic relating to the project here. This one is by Alan Warburton / © BBC / Better Images of AI / Plant / CC-BY 4.0](https://betterimagesofai.org/ImagesAI/WarburtonNature_01_1280x720.jpg)
+
+## 📂 Contents
+
+**_Write a high level outline of the repository contents and structure. What are the main functions and outputs and where can the reader find them? This is not a replacement for informative readmes in sub-directories!_**
+
+## 🛠️ Setup and installation
+
+**Step 1.** Check that you meet the [data science cookiecutter requirements](http://nestauk.github.io/ds-cookiecutter/quickstart). In brief:
+
+Install the following components:
+
+- [gh](https://formulae.brew.sh/formula/gh) - GitHub command line tool.
+- [direnv](https://formulae.brew.sh/formula/direnv#default) - For using environment variables.
+- [git-crypt](https://github.com/AGWA/git-crypt/blob/master/INSTALL.md#installing-on-mac-os-x) - Tool for encryption of sensitive files.
+
+Have a Nesta AWS account, and install and configure your [AWS Command Line Interface](https://docs.aws.amazon.com/polly/latest/dg/setup-aws-cli.html)
+
+**Step 2.** Setup other dependencies.
+
+**_Are there any other dependencies that this project requires?_**
+
+**Step 3.** Install, configure and activate the environment for this project on your machine.
+
+Create a blank cookiecutter conda log file:
+
+```
+$ mkdir .cookiecutter/state
+$ touch .cookiecutter/state/conda-create.log
+```
+
+Run `make install` to configure the development environment:
```

Comment:
```suggestion
Run `make install` to configure the development environment. This will do the following:
```

For someone who does this for the first time, this might be quite cryptic - perhaps it's OK to be a bit more verbose throughout the setup steps.